source/_components/notify.apns.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "APNS"
+description: "Instructions how to add APNS notifications to Home Assistant."
+date: 2015-06-22 10:00
```

Comment:
As long as the the date is in the past, the page get deployed. It would still be nice if the date is close to the date ehrn the page was created.